src/PublicSuffixList.php

```diff
@@ -226,10 +226,10 @@ protected function readCachedPSL($url)
         $cacheFile = $this->getCacheFileName($url);
         if (file_exists($cacheFile)) {
             $cachedTree = file_get_contents($cacheFile);
-            if((int) PHP_VERSION_ID < 70000) {
+            if (PHP_VERSION_ID < 70000) {
```

Comment:
Since we require 7.4 we should drop this if block. We are should always protect with **allowed_classes** for security.